libs/src/sdk/utils/object.ts

```diff
@@ -0,0 +1,19 @@
+// Return properties from a list that the object is missing
+export const objectMissingProperties = (
+  object: Record<string, any>,
+  properties: string[]
+) => {
+  return properties.filter((p) => {
+    return !Object.prototype.hasOwnProperty.call(object, p)
+  })
+}
+
+// Return properties from a list for which the object is missing values
+export const objectMissingValues = (
+  object: Record<string, any>,
+  properties: string[]
+) => {
+  return properties.filter(
+    (p) => !Object.prototype.hasOwnProperty.call(object, p) || object[p] === ''
```

Comment:
```suggestion
    (p) => !object[p]
```

I think what you have here would not include a missing value if it was explicitly set to null or undefined

Reply:
Yeah you're right, this was copied from libs so there may have been a reason for it? But I'm gonna clean this function up

Reply:
oh word